ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -38,66 +37,69 @@ Item {
         height: contentHeight
         objectName: "chatListItems"
         model: root.model
-        spacing: 4
+        spacing: 0
```

Comment:
This, together with turning the delegate into a `Control` and enabling its `verticalPadding`, is here to fully hide a section of items; otherwise it would still have some height due to the spacing between items